app/src/pages/inside/testCaseLibraryPage/testCaseList/utils.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Formats a timestamp in milliseconds to a readable date-time string
+ * @param timestamp - Timestamp in milliseconds
+ * @returns Formatted date string in YYYY-MM-DD HH:MM:SS format
+ */
+export const formatTimestamp = (timestamp: number): string => {
+  const date = new Date(timestamp);
+  const year = date.getFullYear();
+  const month = String(date.getMonth() + 1).padStart(2, '0');
+  const day = String(date.getDate()).padStart(2, '0');
+  const hours = String(date.getHours()).padStart(2, '0');
+  const minutes = String(date.getMinutes()).padStart(2, '0');
+  const seconds = String(date.getSeconds()).padStart(2, '0');
+
+  return `${year}-${month}-${day} ${hours}:${minutes}:${seconds}`;
+};
+
+/**
+ * Formats duration in milliseconds to a human-readable string
+ * @param milliseconds - Duration in milliseconds
+ * @returns Formatted duration string (e.g., "16min", "1h 5min", "2h")
+ */
+export const formatDuration = (milliseconds: number): string => {
+  const minutes = Math.floor(milliseconds / 60000);
+  const hours = Math.floor(minutes / 60);
+
+  if (hours > 0) {
+    const remainingMinutes = minutes % 60;
+    return remainingMinutes > 0 ? `${hours} h ${remainingMinutes} min` : `${hours}h`;
+  }
+  return `${minutes} min`;
+};
+
+/**
+ * Formats a timestamp to relative time (e.g., "2 days ago", "Yesterday", "1 hour ago")
+ * @param timestamp - Timestamp in milliseconds
+ * @returns Formatted relative time string
+ */
+export const formatRelativeTime = (timestamp: number): string => {
```

Comment:
I think we need some library for that. Also, don't forget about localization. @AmsterGet, is there already some mechanism used on the project for such things? I am thinking about `dayjs` or `date-fns`